### PR TITLE
bront_fonts: init at 2015-06-28

### PR DESCRIPTION
**Pull request by GRBurst**
Signed-off-by: GRBurst <GRBurst@protonmail.com>

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/bront/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation { 
+  name = "bront_fonts";
```
Comment (Lassulus) [inline pkgs/data/fonts/bront/default.nix:4]:
```suggestion
  pname = "bront_fonts";
```

**Review by ryantm — CHANGES_REQUESTED** [review]
Please update based on suggestion.

**Comment by SuperSandro2000** [comment]
@GRBurst ping

**Comment by GRBurst** [comment]
Sry all, got totally off my radar 🙈 
Thanks for the suggestion @Lassulus and @ryantm

File: pkgs/data/fonts/bront/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation {
+  pname = "bront_fonts";
+  version = "2015-06-28";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/bront/default.nix:5]:
```suggestion
  version = "unstable-2015-06-28";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**